docs/development/index.rst

```diff
@@ -143,16 +149,18 @@ Not doing so `clobbers the original traceback`_, making it impossible to use
 
 Serialization
 ~~~~~~~~~~~~~
-To ensure the reproducibility of scientific experiments Blocks tries to make
-sure that stopping and resuming training doesn't affect the final results. In
-order to do so it takes a radical approach, serializing the entire training
-state using pickle_. Some things cannot be pickled, so their use should be
-avoided when the object will be pickled as part of the main loop:
+To ensure the reproducibility of scientific experiments, Blocks and Fuel
```

Comment:
This paragraph is about Blocks only, I guess. The pronoun "it" can not refer to Blocks and Fuel.